src/components/graph/menus/network-modification-node-editor.js

```diff
@@ -446,7 +473,47 @@ const NetworkModificationNodeEditor = () => {
             studyUuid,
             currentTreeNode?.id,
             [...selectedItems.values()].map((item) => item.uuid)
-        ).then();
+        )
+            .then()
+            .catch((errmsg) => {
+                console.error('doDeleteModification error:', errmsg);
+                snackError('', 'errDeleteModificationMsg');
+            });
+    };
+
+    const doCopyModification = () => {
+        // just memorize the list of selected modifications
+        let newCopySet = new Set();
+        selectedItems.forEach((item) => {
+            if (item.type !== 'GROOVY_SCRIPT') {
+                newCopySet.add(item.uuid);
+            }
+        });
+        setCopiedModifications(newCopySet);
+    };
+
+    const doPasteModification = () => {
+        duplicateModifications(
+            studyUuid,
+            currentTreeNode.id,
+            Array.from(copiedModifications.keys())
+        )
+            .then((modificationsInFailure) => {
+                if (modificationsInFailure.length > 0) {
+                    console.warn(
+                        'Modifications not pasted:',
+                        modificationsInFailure
+                    );
+                    snackWarning(
+                        modificationsInFailure.length,
```

Comment:
The message is a number ?

Reply:
the message id is _warnDuplicateModificationMsg_, taking one param ; the nb of failure.
result example:
```
Number of modifications not copied :
2
```